clients/bmp_client.go

```diff
@@ -79,3 +79,181 @@ func (bc *bmpClient) SlPackages() (SlPackagesResponse, error) {
 
 	return response, nil
 }
+
+func (bc *bmpClient) Stemcells() (StemcellsResponse, error) {
+	path := fmt.Sprintf("%s/%s", bc.url, "/stemcells")
+	responseBytes, errorCode, err := bc.httpClient.DoRawHttpRequest(path, "GET", &bytes.Buffer{})
+	if err != nil {
+		errorMessage := fmt.Sprintf("bmp: could not calls /stemcells on BMP server, error message '%s'", err.Error())
+		return StemcellsResponse{}, errors.New(errorMessage)
+	}
+
+	if slcommon.IsHttpErrorCode(errorCode) {
+		errorMessage := fmt.Sprintf("bmp: could not call /stemcells on BMP server, HTTP error code: '%d'", errorCode)
+		return StemcellsResponse{}, errors.New(errorMessage)
+	}
+
+	response := StemcellsResponse{}
+
+	err = json.Unmarshal(responseBytes, &response)
+	if err != nil {
+		errorMessage := fmt.Sprintf("bmp: failed to decode JSON response, err message '%s'", err.Error())
+		return StemcellsResponse{}, errors.New(errorMessage)
+	}
+
+	return response, nil
+}
+
+func (bc *bmpClient) SlPackageOptions(packageId string) (SlPackageOptionsResponse, error) {
+	path := fmt.Sprintf("%s/sl/packages/%s/options", bc.url, packageId)
+
```

Comment:
Remove new line here as well. Same rationale as before. Please be consistent throughout. I won't repeat that comment again. Thanks.